INSTALL.md

```diff
@@ -1,11 +1,47 @@
+## Installation requirements on ubuntu
+
 WCIVF requires Python 3.
 
 To install:
 
     sudo apt-get install python3-dev libpq-dev libjpeg-dev redis-server
     pip install -r requirements/local.txt
 
-Create a Postgres database as detailed below, then:
+
+## Installation requirements on macOS
+
+Use homebrew to install the following dependencies:
+
+    brew install postgresql libjpeg-dev postgis
```

Comment:
FWIW, I always used to use https://postgresapp.com/ when I was on a mac, as it resolved a tonne of problems installing with Brew. Also, multiple version support was super useful.

Also, I'm not 100% sure, but I don't think postgis is a requirement for this project any more. We use it elsewhere though, so good to have it installed.